src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1219,16 +1242,17 @@ private void NotifyFeaturesAboutValueChange(DatabaseRecord record, long index)
                 Monitor.TryEnter(_clusterLocker, TimeSpan.FromSeconds(5), ref taken);
                 try
                 {
-                    if (CanSkip(record.DatabaseName, index))
+                    if (CanSkipValueChange(record.DatabaseName, index))
                         return;
 
                     if (taken == false)
                         continue;
                 
-                    LastDatabaseRecordIndex = index;
                     DatabaseShutdown.ThrowIfCancellationRequested();
                     SubscriptionStorage?.HandleDatabaseRecordChange(record);
                     EtlLoader?.HandleDatabaseValueChanged(record);
+
+                    LastDatabaseRecordChangeIndex = index;
```

Comment:
Shouldn't this be `LastValueChangeIndex` ?

Reply:
You are absolutely correct! I will add a test for this.